react/week3/src/components/todo.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+
+function Todo({ todo, id, todoDone, todoRemove, todoUpdate }) {
+  console.log(todo);
+
+  const [editedTodo, setEditedtodo] = useState({ description: todo.description, deadline: todo.deadline });
+  const [iscmdEdit, setcmdEdit] = useState(false);
+  //console.log(index, todo.description);
+  // console.log(index,editedTodo.description);
+  return (
+
+    <div className='todo' style={{ textDecoration: todo.isComplete ? 'line-through' : '' }}>

```

Comment:
This return part is very hard to read. Try to break it up into smaller properly names parts.